modules/tide_content_collection/css/content-collection-ui-widget.css

```diff
@@ -0,0 +1,10 @@
+div[class*=tabs-advanced-display-sort-criteria]{
+  display: inline-block;
+  margin-right: 0.75em;
+}
+
+.js .field--widget-paragraphs tr[data-drupal-selector*=tabs-advanced-display-sort-elements] td,
```

Comment:
@joshua-salsadigital please ensure that the CSS selector only *targets* the content collection widget, not all paragraph widgets.

Reply:
Added a selector to target the parent custom widget.